### PR TITLE
Mobile first

### PR DESCRIPTION
**Pull request by MariamOsman11**
# What I did
-Understand how to parse a Figma design to create a UI.
-Use Flexbox to place elements in the page.
-Use CSS Grid to place elements in the page.
-Build a personal portfolio site.

### CODE REVIEW

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @MariamOsman11 🙋‍♀️

Well done! 🥇🥇 I am impressed by your style and attention to detail. You deserve an applause 👏👏 I know you worked hard to implement this project, and I really appreciate the effort you put in!"

**_However, there are a few things that still need to be fixed. Don't worry, these are very minor changes and can easily be addressed. But overall, you are doing great!_** 💪 🏁

so_close

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Well done following the given Figma design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

However, there is always room for improvement.

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

- [ ] The issues link is not working right now 🔗 I'm getting a **_File not found error_**. Please fix this as well. I know you can do that 💪

image


_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -238,14 +238,14 @@ <h4 class="langs-text">CSS</h4>
...
+              <img src="images/Union2.svg" alt="arrow" />
             </div>
             <hr class="new1" />
           </div>
           <div>
             <div class="skill">
               <h4 class="skill-title">Skills</h4>
-              <img src="images/Union2.png" alt="arrow" />
+              <img  src="images/Union2.svg" alt="arrow" />
             </div>
```
Comment (tugsem) [inline index.html]: - [ ] As the prev reviewer requested, kindly wrap the frameworks and skills into an `ul` tag for semantic reasons.

File: mobile-style.css [hunk 2]
```diff
@@ -236,15 +236,15 @@ header {
...
   padding: 12px;
   margin-bottom: 26px;
 }
 
 .card-btn:hover {
   color: #fff;
-  background-color: #396df2;
+  background-color: #6070ff;
   transition: 1s ease-in-out;
 }
```
Comment (tugsem) [inline mobile-style.css]: - [ ] Good job adding the right color as requested by the reviewer, but she also mentions that the same thing for this part of code. Kindly fix it.
image

**Review by tugsem — CHANGES_REQUESTED** [review]
## STATUS: REQUIRED CHANGES ♻️
Hello @MariamOsman11 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

 ## Highlights 🚀
✔️Good job on applying most of the changes

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @MariamOsman11  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ No linter errors 👍 
✔ Descriptive PR 👍 
✔ Made suggested changes👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**